camel/toolkits/twitter_toolkit.py

```diff
@@ -0,0 +1,524 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import datetime
+import os
+from http import HTTPStatus
+from http.client import responses
+from typing import List, Optional, Tuple, Union
+
+import requests
+
+from camel.toolkits import OpenAIFunction
+from camel.toolkits.base import BaseToolkit
+
+TWEET_TEXT_LIMIT = 280
+
+
+def get_twitter_api_key() -> Tuple[str, str]:
```

Comment:
nit: just to unify the code style as what you did for `class WeatherToolkit`, we can also move this function within the class, use `def _get_twitter_api_key` representing it's internal function